recordings/basic-setup/run.sh

```diff
@@ -0,0 +1,309 @@
+crd/scyllaclusters.scylla.scylladb.com \
+crd/nodeconfigs.scylla.scylladb.com \
+crd/scyllaoperatorconfigs.scylla.scylladb.com \
+crd/scylladbmonitorings.scylla.scylladb.com
+EOF
+run <<'EOF'
+kubectl -n=scylla-operator rollout status --timeout=10m deployment.apps/{scylla-operator,webhook-server}
+EOF
+
+run <<< "# To run ScyllaClusters on local disk (NVMes in production) we need to setup our storage."
+
+run <<< "# We need to setup our local disks into a XFS directory, that will be used by ScyllaDB local CSI driver to provision PVCs."
+run <<< "# We will also enable node tuning."
+
+run <<< "# Create a NodeConfig to setup RAID, filesystem and tuning."
+run <<< "# Note that this part depend on your infra, disk names or other properties."
+run <<< "# Be sure to double check the NodeConfig and adjust it, if needs be."
+run <<'EOF'
+yq -e eval --colors '.' ./examples/gke/nodeconfig-alpha.yaml | head -n 25
+EOF
+sleep-extra 10
+run <<'EOF'
+kubectl apply --server-side -f=./examples/gke/nodeconfig-alpha.yaml
+EOF
+
+run <<< "# Wait for the NodeConfig to apply changes to the Kubernetes nodes."
+run <<'EOF'
+kubectl wait --for='condition=Reconciled' --timeout=10m nodeconfigs.scylla.scylladb.com/cluster
+EOF
+# TODO: Extend waiting for the NodeConfig when we fix https://github.com/scylladb/scylla-operator/issues/1557
+
+run <<< "# Install ScyllaDB local CSI driver that will provision storage for PersistentVolumeClaims."
+run <<< "# The ScyllaDB local CSI driver is configured to use folder /mnt/persistent-volumes that was created by the NodeConfig."
+run <<'EOF'
+kubectl -n local-csi-driver apply --server-side -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/v0.2.0-rc.0/{deploy/kubernetes/local-csi-driver/{00_namespace,10_csidriver,10_driver_serviceaccount,10_provisioner_clusterrole,20_provisioner_clusterrolebinding},example/storageclass_xfs}.yaml
+kubectl create -f=https://raw.githubusercontent.com/scylladb/k8s-local-volume-provisioner/v0.2.0-rc.0/deploy/kubernetes/local-csi-driver/50_daemonset.yaml --dry-run=client -o yaml | yq -e '.spec.template.spec.nodeSelector += {"scylla.scylladb.com/node-type": "scylla"}' | kubectl -n=local-csi-driver apply --server-side -f -
+kubectl -n=scylla-operator rollout status --timeout=10m deployment.apps/{scylla-operator,webhook-server}
+EOF
+run <<'EOF'
+kubectl -n=local-csi-driver rollout status --timeout=10m daemonset.apps/local-csi-driver
+EOF
+
+run <<< "# Because all the following manifests use default storage class, we'll set scylladb-local-xfs as the default one, for simplicity."
+run <<< "# As there can be only one default storage class, make sure to unset the previous default one, if it existed in your setup."
+run <<< "# This can be also controlled per ScyllaCluster."
+run <<'EOF'
+kubectl patch storageclass scylladb-local-xfs -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+EOF
+sleep-extra 5
+
+run <<< "# Optionally, deploy Scylla Manager"
```

Comment:
```suggestion
run <<< "# Optionally, deploy Scylla Manager."
```
nit, applies globally (some have it, some don't)

Reply:
done